usaspending_api/awards/models.py

```diff
@@ -384,12 +405,27 @@ def get_default_fields():
         default_fields = AwardAction.get_default_fields()
         return default_fields + [
             "type",
+            "type_description",
             "cost_or_pricing_data",
+            "type_of_contract_pricing",
+            "type_of_contract_pricing_description",
             "naics",
             "naics_description",
             "product_or_service_code"
         ]
 
+    def get_pricing_type_description(self):
+        description = [item for item in CONTRACT_PRICING_TYPES if item[0] == self.type_of_contract_pricing]
+        if len(description) == 0:
+            return "Unknown Type"
```

Comment:
Back to the earlier question of whether or not contract pricing type is relevant/expected on all contract records. If it's not, maybe we should return `None` here. If it *is* something we expect on every record, should we also use some kind of "unknown" code in `type_of_contract_pricing` for consistency?